src/cli/server/server.rs

```diff
@@ -52,11 +52,14 @@ impl Server {
     }
 
     /// Starts the server in its own multithreaded Runtime
-    pub async fn fork_start(self) -> Result<()> {
+    pub async fn fork_start(self, usage_tracking: bool) -> Result<()> {
         let runtime = tokio::runtime::Builder::new_multi_thread()
             .worker_threads(self.config_module.deref().server.get_workers())
             .enable_all()
             .build()?;
+        if usage_tracking {
+            let _ = tailcall_events::EventRequest::alive_event_poll(&runtime).await;
```

Comment:
Drop this code. If tracking is enabled (decided by tailcall_tracker, it should automatically keep polling).